openedx/features/course_experience/__init__.py

```diff
@@ -22,49 +23,31 @@ class DefaultTrueWaffleFlagNamespace(WaffleFlagNamespace):
         and refactor/fix any tests that shouldn't be removed.
 
     """
-    def is_flag_active(self, flag_name, check_before_waffle_callback=None):
+    def _get_flag_active_request(self, namespaced_flag_name, request):
```

Comment:
Is this a method that subclasses are meant to override?  If so, is `_` prefix inappropriate?  
Also, I need to review more deeply to see this method and understand how and if this change makes sense.

Reply:
> Is this a method that subclasses are meant to override?

Yes.

> If so, is _ prefix inappropriate?

I don't think so. There is only a linting problem when overriding methods prefixed with double underscore `__` (unless I'm mistaken?).

Reply:
This makes sense.  I still feel new to Python conventions.  Thanks.